vic/drivers/shared_all/src/history_metadata.c

```diff
@@ -0,0 +1,1482 @@
+    strcpy(out_metadata[OUT_SOIL_TNODE_WL].varname, "OUT_SOIL_TNODE_WL");
+    strcpy(out_metadata[OUT_SOIL_TNODE_WL].long_name, "soil_tnode_wl");
+    strcpy(out_metadata[OUT_SOIL_TNODE_WL].standard_name, "soil_temperature");
+    strcpy(out_metadata[OUT_SOIL_TNODE_WL].units, "C");
+    strcpy(out_metadata[OUT_SOIL_TNODE_WL].description,
+           "soil temperature for each soil thermal node in the wetland");
+
+    /* soil temperature flag for each soil thermal node */
+    strcpy(out_metadata[OUT_SOILT_FBFLAG].varname, "OUT_SOILT_FBFLAG");
+    strcpy(out_metadata[OUT_SOILT_FBFLAG].long_name, "soilt_fbflag");
+    strcpy(out_metadata[OUT_SOILT_FBFLAG].standard_name,
+           "soil_temperature_flag");
+    strcpy(out_metadata[OUT_SOILT_FBFLAG].units, "1");
+    strcpy(out_metadata[OUT_SOILT_FBFLAG].description,
+           "soil temperature flag for each soil thermal node");
+
+    /* average surface temperature [C] */
+    strcpy(out_metadata[OUT_SURF_TEMP].varname, "OUT_SURF_TEMP");
+    strcpy(out_metadata[OUT_SURF_TEMP].long_name, "surf_temp");
+    strcpy(out_metadata[OUT_SURF_TEMP].standard_name, "surface_temperature");
+    strcpy(out_metadata[OUT_SURF_TEMP].units, "C");
+    strcpy(out_metadata[OUT_SURF_TEMP].description,
+           "average surface temperature");
+
+    /* surface temperature flag */
+    strcpy(out_metadata[OUT_SURFT_FBFLAG].varname, "OUT_SURFT_FBFLAG");
+    strcpy(out_metadata[OUT_SURFT_FBFLAG].long_name, "surft_fbflag");
+    strcpy(out_metadata[OUT_SURFT_FBFLAG].standard_name,
+           "surface_temperature_flag");
+    strcpy(out_metadata[OUT_SURFT_FBFLAG].units, "1");
+    strcpy(out_metadata[OUT_SURFT_FBFLAG].description,
+           "surface temperature flag");
+
+    /* Tcanopy flag */
+    strcpy(out_metadata[OUT_TCAN_FBFLAG].varname, "OUT_TCAN_FBFLAG");
+    strcpy(out_metadata[OUT_TCAN_FBFLAG].long_name, "tcan_fbflag");
+    strcpy(out_metadata[OUT_TCAN_FBFLAG].standard_name,
+           "canopy_temperature_flag");
+    strcpy(out_metadata[OUT_TCAN_FBFLAG].units, "1");
+    strcpy(out_metadata[OUT_TCAN_FBFLAG].description, "Tcanopy flag");
+
+    /* depth of thawing fronts [cm] for each thawing front */
+    strcpy(out_metadata[OUT_TDEPTH].varname, "OUT_TDEPTH");
+    strcpy(out_metadata[OUT_TDEPTH].long_name, "tdepth");
+    strcpy(out_metadata[OUT_TDEPTH].standard_name, "depth_of_thawing_fronts");
+    strcpy(out_metadata[OUT_TDEPTH].units, "cm");
+    strcpy(out_metadata[OUT_TDEPTH].description,
+           "depth of thawing fronts for each thawing front");
+
+    /* Tfoliage flag */
+    strcpy(out_metadata[OUT_TFOL_FBFLAG].varname, "OUT_TFOL_FBFLAG");
+    strcpy(out_metadata[OUT_TFOL_FBFLAG].long_name, "tfol_fbflag");
+    strcpy(out_metadata[OUT_TFOL_FBFLAG].standard_name,
+           "foliage_temperature_flag");
+    strcpy(out_metadata[OUT_TFOL_FBFLAG].units, "1");
+    strcpy(out_metadata[OUT_TFOL_FBFLAG].description,
+           "depth of thawing fronts for each thawing front");
```

Comment:
this description is for the above variable, I think you want `"flag for temperature of foliage"`